domain/build.gradle

```diff
@@ -1,24 +1,31 @@
-apply plugin: 'info.solidsoft.pitest'
```

Comment:
The pitest plugin should be applied as it is needed for the mutation testing to work

Reply:
Confirmed